query/rdf_result_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build integration || cloud
+//go:build integration || cloud || upgrade
```

Comment:
keeping track of these tags will become difficult, is there another way to initialize the test spin up?

Reply:
I am hoping we won't have any more tags. I am also hoping that eventually, integration2 will also be renamed to integration.